src/compas_fab/robots/constraints.py

```diff
@@ -320,6 +345,79 @@ def copy(self):
         cls = type(self)
         return cls(self.joint_name, self.value, self.tolerance_above, self.tolerance_below, self.weight)
 
+    @classmethod
+    def joint_constraints_from_configuration(cls, configuration, tolerances_above, tolerances_below):
```

Comment:
We are in the `Constraint` class, so I thought if we simply have a `Constraint.from_configuration`, it will misled people in thinking that it returns one Constraint. In reality it returns a list.